playbooks/provisioning/aws/cloudformation/cloudformation_setup.yml

```diff
@@ -1,61 +0,0 @@
-# vim: set ft=ansible:
----
-- name: 'Bootstrapping or Refreshing Environment'
-  hosts: localhost
-  connection: local
-  sudo: no
-  gather_facts: no
-  vars_files:
-  - vars.yml
-  vars:
-    vpc_subnet_azs: "{{ lookup('ec2_zones_by_region', ec2_region) }}"
-    r53_subdomain: "{{ cluster_id }}.{{ env_id }}.{{ r53_zone }}"
-  tasks:
-  - name: 'Validating options'
-    fail:
-      msg: required values not set
```

Comment:
The cloudformation stuff was a previous attempt, I'm working on getting that removed in a separate PR to clean this PR up.